nightly/index.html

```diff
@@ -0,0 +1,218 @@
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.min.js" integrity="sha384-7VPbUDkoPSGFnVtYi0QogXtr74QeVeeIs99Qfg5YCF+TidwNdjvaKZX19NZ/e6oz" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.9.0/js/bootstrap-datepicker.min.js" integrity="sha512-T/tUfKSV1bihCnd+MxKD0Hm1uBBroVYBOYSk1knyvQ9VyZJpc/ALb4P0r6ubwVPSGB2GvjeoMAJJImBG12TiaQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-icons/1.8.1/font/bootstrap-icons.min.css" integrity="sha512-Oy+sz5W86PK0ZIkawrG0iv7XwWhYecM3exvUtMKNJMekGFJtVAhibhRPTpmyTj8+lJCkmWfnpxKgT2OopquBHA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <div class="container">
+      <h1 class="text-center">Amazon Corretto nightly build downloads</h1>
+      <p>
+        Amazon Corretto provides nightly builds for evaluation and testing purposes. These builds may contain bugs or breaking changes in functionality and are not supported in production environments.
+        For the latest supported releases please visit <a href="https://corretto.aws">Corretto.aws</a> for download and installation instructions.
+      </p>
+      <div class='input-group date' id='datepicker' style="justify-content: center"><p>Showing builds for
+        <input type="text" value="" size="12">
+        <span class="input-group-addon"><span class="bi bi-calendar3"></span></span>
+        </p>
+    </div>
+
+    <div class="container">
+      <p>
+      <div class="row">
+        <div class="col">
+          <select id="versionSelect" class="form-select required" onchange="onSelectChange()"> <option selected>All Versions</option></select>
+        </div>
+        <div class="col">
+          <select id="platformSelect" class="form-select required" onchange="onSelectChange()"> <option selected>All Platforms</option></select>
+        </div>
+        <div class="col">
+          <select id="architectureSelect" class="form-select required" onchange="onSelectChange()"> <option selected>All Architectures</option></select>
+        </div>
+      </div>
+      </p>
+    </div>
+
+    <div class="container" style="background-color:white;color: black;"></div>
+    <table class="table table-bordered table-striped">
+      <thead>
+        <tr>
+          <th scope="col">Version</th>
+          <th scope="col">Branch</th>
+          <th scope="col">Commit</th>
+          <th scope="col">Debug Level</th>
+          <th scope="col">Platform</th>
+          <th scope="col">Architecture</th>
+          <th scope="col">Artifacts</th>
+        </tr>
+      </thead>
+      <tbody id="target">
+      </tbody>
+    </table>
+    <script type="text/javascript">
```

Comment:
This is enough JS that I'm wondering if it's worth extracting to its own file and minifying

Reply:
It is only about 6k of JS right now. I would like to wait to pull it out and minify until after this is released as this is much easier to debug.